ch_tools/chadmin/cli/data_store_group.py

```diff
@@ -513,9 +516,6 @@ def detach_partition(ctx: Context, table_partition: TablePartition) -> bool:
     Run DETACH the partition.
     """
 
-    logging.debug(
```

Comment:
why?(

Reply:
query is being already written by query_with_retry.